ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -163,6 +163,17 @@ public IncrementalLoadTasksBuilder(String dbName, String loadPath,
       taskChainTail.addDependentTask(updateIncPendTask);
       taskChainTail = updateIncPendTask;
 
+      Database dbToLoadIn = hive.getDatabase(dbName);
+      if (dbToLoadIn != null && !ReplUtils.isTargetOfReplication(dbToLoadIn)) {
+        Map<String, String> props = new HashMap<>();
+        props.put(ReplUtils.TARGET_OF_REPLICATION, "true");
```

Comment:
Can you please check if this is not overriding an existing property?

Reply:
It just appends this prop to existing db properties.